data/curated/new_submission/billboard.md

```diff
@@ -0,0 +1,107 @@
+|:----------------------------------------------|:-----|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+|Song                                           |NULL  |Song Name                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
```

Comment:
All of these are showing as class `NULL`, and the displayed variable names haven't gone through `clean_names()`. I haven't downloaded the billboard.csv to check, but is it possible you saved BEFORE that final cleaning?

I recommend renaming this file then copy/pasting to preserve the `description` column (or you could load the md into a data.frame, join, and re-write if you're feeling fancy, but I can't remember the code to do all that off the top of my head).